Pipfile

```diff
@@ -36,6 +36,7 @@ pytest-randomly = "*"
 sentry-sdk = {extras = ["flask"], version = "*"}
 ray = {version = "==1.7.0", extras = ["default"]}
 celery = {version = "==5.1.2", extras = ["redis"]}
+"stomp.py" = "*"
```

Comment:
It's better to have fixed version in order to avoid suprises when installing new packages